src/chrome/content/rules/Cheapshark.com.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	Nonfunctional hosts in *.cheapshark.com:
+
+	h: http redirect
```

Comment:
They don't redirect to `http`on the same domain, so they can be added as target.

Reply:
Okay my bad, I did this before I got your message about http redirects. I think I understand now. Thanks J0WI